tests/unit/dojson/test_dojson_hep.py

```diff
@@ -2203,3 +2203,133 @@ def test_report_numbers_from_two_037__a():
 
     assert expected == result['037']
 
+
+def test_keywords_from_695__a_2():
+    schema = load_schema('hep')
+    subschema = schema['properties']['keywords']
+
+    snippet = (
+        '<datafield tag="695" ind1=" " ind2=" ">'
+        '  <subfield code="a">REVIEW</subfield>'
+        '  <subfield code="2">INSPIRE</subfield>'
+        '</datafield>'
+    )  # record/200123
+
+    expected = [
+        {
+            'keyword': 'REVIEW',
+            'classification_scheme': 'INSPIRE',
+        },
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['keywords'], subschema) is None
+    assert expected == result['keywords']
+
+    expected = [
+        {
+            'a': 'REVIEW',
+            '2': 'INSPIRE',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['695']
+
+
+def test_energy_ranges_from_695__2_e():
+    schema = load_schema('hep')
+    subschema = schema['properties']['energy_ranges']
+
+    snippet = (
+        '<datafield tag="695" ind1=" " ind2=" ">'
+        '  <subfield code="2">INSPIRE</subfield>'
+        '  <subfield code="e">7</subfield>'
+        '</datafield>'
+    )  # record/1124337
+
+    expected = [7]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['energy_ranges'], subschema) is None
+    assert expected == result['energy_ranges']
+
+    expected = [
+        {
+            '2': 'INSPIRE',
+            'e': 7,
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['695']
+
+
+def test_keywords_from_multiple_695__a_2():
+    schema = load_schema('hep')
+    subschema = schema['properties']['keywords']
+
+    snippet = (
+        '<datafield tag="695" ind1=" " ind2=" ">'
+        '  <subfield code="a">programming: Monte Carlo</subfield>'
+        '  <subfield code="2">INSPIRE</subfield>'
+        '</datafield>'
+        '<datafield tag="695" ind1=" " ind2=" ">'
+        '  <subfield code="a">electron positron: annihilation</subfield>'
+        '  <subfield code="2">INSPIRE</subfield>'
+        '</datafield>'
+    )  # record/363605
+
+    expected = [
+        {
+            'keyword': 'programming: Monte Carlo',
+            'classification_scheme': 'INSPIRE',
+        },
+    ]
```

Comment:
Watch out! Because you are not wrapping the two `<datafield>` tags with a `<record>` tag `create_record` only outputs the first.